txjson_node_test.go

```diff
@@ -109,7 +109,7 @@ func TestTxJSON_Node_MarshallJSON(t *testing.T) {
 			"value": 0,
 			"n": 0,
 			"scriptPubKey": {
-				"asm": "OP_FALSE OP_RETURN 48656c6c6f",
+				"asm": "0 OP_RETURN 48656c6c6f",
```

Comment:
Has OP_FALSE been deprecated in place of just showing 0?

Reply:
Although this is not my code, I tried changing it to OP_FALSE and the test failed.  This may need extra investigation if someone wants to take a look.